test/datetime.test.js

```diff
@@ -17,6 +17,7 @@ describe('MySQL datetime handling', function() {
     married: Boolean,
     dob: {type: 'DateString'},
     createdAt: {type: Date, default: Date},
+    lastLogon: {type: Date, precision: 3, default: Date},
```

Comment:
hi @joostdebruijn from the login on line#900, a property with `date` type won't execute `dateOptions()` which appends precision config, can you take a look?